node-10-error.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+const path = require('path');
+const { parse } = require('./dist/lexer.cjs.js');
```

Comment:
This should be `./dist/lexer.cjs`.

Note the extension is `.cjs` not `.js` because this package has a package.json `"type": "module"` field so a `.cjs` extension is needed in line with the latest --experimental-modules advice.